app/models/supported_ssg.rb

```diff
@@ -3,10 +3,11 @@
 require 'yaml'
 
 # rubocop:disable Metrics/BlockLength
-SupportedSsg = Struct.new(:id, :package, :version, :profiles,
+SupportedSsg = Struct.new(:id, :package, :version, :upstream_version, :profiles,
                           :os_major_version, :os_minor_version,
                           keyword_init: true) do
   self::SUPPORTED_FILE = Rails.root.join('config/supported_ssg.yaml')
+  self::OLDEST_UPSTREAM_VERSION = '0.1.24'
```

Comment:
We could mark the versions that don't have any upstream counterpart in the config itself to avoid using this constant, perhaps something like `upstream_version: ''` or `upstream_version: 'N/A'`, `upstream_version: 0.0.0` :man_shrugging: What do you think?

Reply:
Great suggestion.